### ISSUE TITLE
FOTD April 17, 1997: Dewdrop Lattice doesn't render correctly

### ISSUE THREAD

**Issue by LegalizeAdulthood** [LegalizeAdulthood/iterated-dynamics#57]
Correct image:

![FD170497](https://github.com/LegalizeAdulthood/iterated-dynamics/assets/4270863/20518894-5f02-4697-8bf3-8b2862fb8cce)

**Comment by LegalizeAdulthood** [comment]
ManPWin is unable to render this correctly, implying a C code formula parser bug present when ManPWin forked.

**Comment by PaulTheLionHeart** [comment]
ManpWIN and ID give the same result. The only common denominator is the C++ version of the formula parser

**Comment by PaulTheLionHeart** [comment]
ManpWIN image:
![image](https://github.com/user-attachments/assets/fd0bc0dd-206c-43e0-b9fa-9eadf451453f)

**Comment by PaulTheLionHeart** [comment]
Can we use something like floarexp? I use it when the exponent of a double is outside 1e-308 -> 1e308.

**Comment by PaulTheLionHeart** [comment]
Yes, true. And you have a copy of my ManpWIN repository. Much easier. I haven't updated it with Pert/BLA and the new multi-threading code yet. I am also experimenting with templates to help reduce the duplication between Double, DD, QD, mpfr and make adding new fractals easier.

**Comment by LegalizeAdulthood** [comment]
They added double double support to boost.multiprecision in boost 1.89.  I should be able to use that to implement 4-, 6-, or 8-double back ends.

**Comment by LegalizeAdulthood** [comment]
It's on the [roadmap for 3.0](https://github.com/LegalizeAdulthood/iterated-dynamics/wiki#iterated-dynamics-30) but might show up sooner `:)`
